templates/macros.html

```diff
@@ -2,6 +2,14 @@
     Makes the appropriate JS imports for highlighting
     * `languages` An array of strings where each is a valid highlight.js language
 #}
+{% macro active_link(expected, href, text) %}
+    <li class="pure-menu-item">
+        <a class="pure-menu-link {% if active_tab == expected %} pure-menu-active {% endif %}" href="{{ href }}">
+        {{ text }}
+        </a>
+    </li>
+{% endmacro active_link %}
```

Comment:
Documentation for the `active_link` macro
```suggestion
{#
    Creates a list entry for active tabs. When the active tab is the same as `expected`, it will show the current tab as active.
    * `expected` A string that represents the current tab, when `active_tab == expected` the current will be shown as active
    * `href` A string used as the tab's link
    * `text` A string used as the tab's text
#}
{% macro active_link(expected, href, text) %}
    <li class="pure-menu-item">
        <a class="pure-menu-link {% if active_tab == expected %} pure-menu-active {% endif %}" href="{{ href }}">
            {{ text }}
        </a>
    </li>
{% endmacro active_link %}
```